service/src/LevelZeroIOGroup.cpp

```diff
@@ -428,6 +444,13 @@ namespace geopm
                                     GEOPM_DOMAIN_GPU_CHIP,
                                     Agg::expect_same,
                                     string_format_double
+                                    }},
+                               {M_NAME_PREFIX + "GPU_CORE_PERFORMANCE_FACTOR_CONTROL", {
+                                    "Performance Factor",
```

Comment:
```suggestion
                                    "Expresses a trade-off between energy provided to the GPU compute hardware and the supporting units",
```

Please put this and the default value in a man page.  Also some general guidance about what setting numbers closer to 0 means vs. numbers closer to 1.